src/system/dlt-system-syslog.c

```diff
@@ -165,6 +165,7 @@ void start_syslog(DltSystemConfiguration *conf)
 			DLT_STRING("dlt-system-syslog, start syslog"));
 	static pthread_attr_t t_attr;
 	static pthread_t pt;
+	pthread_attr_init(&t_attr);
```

Comment:
Having t_attr uninitialized definitely seems like a bug so I agree with fixing it, but:

- pthread_create() man page says you can instead pass NULL to get the default attributes (which is what this also does, since t_attr is not modified after init call).

- pthread_attr_init() man page says you should destroy t_attr using pthread_attr_destroy() when it is no longer used, which is not done here.

For simplicity, I'd go with NULL.